app/component/menu/menu.scss

```diff
@@ -0,0 +1,62 @@
+@import "_theme";
+
+.app-menu {
+  position: relative;
+  z-index: 101;
+
+  .fa-bars {
+  position: fixed;
+  z-index: 110;
+  border: thin solid white;
+  border-radius: 5%;
+
+  left: 3%;
+  top: 2%;
+  padding: .5% .5% .5% .5%;
+
+  color: $app-lightgreen;
+  font-size: 3em;
+
+    &:hover {
+      color: $app-default;
+      background-color: white;
+    }
+  }
+
+  @media (min-width: 900px){
+    .mobileMenu{
+      display: none;
+    }
+  }
+
+  @media (max-width: 899px){
+    .desktopMenu{
+      display: none;
+    }
+  }
+}
+
+
+.menu-exit.ng-leave {
+  animation: my_fade_animation 0.2s linear;
+  * {
+    animation: my_fade_animation 0.2s linear;
+  }
+}
+
+
+.menu-exit.ng-enter {
+  animation: my_fade_animation 0.2s linear reverse;
+  * {
+    animation: my_fade_animation 0.2s linear reverse;
+  }
+}
+
+@keyframes my_fade_animation {
```

Comment:
you might wat to move each keyframe animation into its own sass partial
for example 
app/scss/animation/_my_fade_animation.scss